web/src/user_groups.ts

```diff
@@ -300,7 +300,7 @@ export function is_setting_group_empty(setting_group: GroupSettingValue): boolea
 }
 
 export function get_user_groups_of_user(user_id: number): UserGroup[] {
-    const user_groups_realm = get_realm_user_groups();
+    const user_groups_realm = get_realm_user_groups(true);
```

Comment:
This function is used in other places as well like for rendering groups in user profile modal. I don't think we want to show deactivated groups there. Have you checked if that works correctly or not?